modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/service/impl/CommerceCurrencyLocalServiceImpl.java

```diff
@@ -64,12 +64,15 @@
 import org.osgi.framework.BundleContext;
 import org.osgi.framework.FrameworkUtil;
 import org.osgi.framework.ServiceRegistration;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Andrea Di Giorgi
  * @author Marco Leo
  * @author Alessio Antonio Rendina
  */
+@Component(enabled = false, service = AopService.class)
```

Comment:
@JanisZhang For now, please add `property = "model.class.name=xxx"` to the component annotation, like other local services e.g. `BlogsEntryLocalServiceImpl`. If we get "no" answer from the Slack thread, then we can remove it.

Reply:
@dantewang OK